tests/test_optics.py

```diff
@@ -1,55 +1,32 @@
 import pytest
 
 
-def test_sunrise_sunset():
+def test_sunrise_sunset_skyfield():
     """
     Tests if sunrise/sunset:
         1. can run
         2. output is the right shape
         3. if the output is correct-ish
     """
     import numpy as np
+    import pandas as pd
 
-    import glidertools as gt
+    from glidertools.optics import sunset_sunrise
 
     time = [
         np.datetime64("2000-01-01"),
         np.datetime64("2000-01-02"),
         np.datetime64("2000-01-03"),
     ]
-    lat = -35, 35, 45
+    lat = -35, 75, 45
```

Comment:
See below: I was wondering why not to use 80 (as in the previously failing test)?